### PR TITLE
[benchmarks] Print % of compute compiled with TVM. Useful for debugging

### PR DESCRIPTION
**Pull request by bwasti**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#35 [benchmarks] Print % of compute compiled with TVM. Useful for debugging**

### CODE REVIEW

File: test/benchmarks.py [hunk 1]
```diff
@@ -42,7 +42,15 @@ def benchmark(model, csv_file, input_fn=genImage, iters=100, warmup=10):
...
         tvm_time = time.time() - start
-        print("Done benchmarking TVM")
+        with torch.autograd.profiler.profile() as prof:
+          _ = trace_tvm(*inputs)
+        tvm_profiled_time = 0
+        total_profiled_time = 0
+        for p in prof.key_averages():
+          total_profiled_time += int(p.cpu_time)
+          if p.key == "TVM":
+            tvm_profiled_time = int(p.cpu_time)
```
Comment (wanchaol) [inline test/benchmarks.py]: I think you missed a `+` here.

Comment (bwasti) [reply test/benchmarks.py]: done :)

File: test/benchmarks.py [hunk 2]
```diff
@@ -42,7 +42,15 @@ def benchmark(model, csv_file, input_fn=genImage, iters=100, warmup=10):
         for _ in range(iters):
           _ = trace_tvm(*inputs)
         tvm_time = time.time() - start
-        print("Done benchmarking TVM")
+        with torch.autograd.profiler.profile() as prof:
+          _ = trace_tvm(*inputs)
+        tvm_profiled_time = 0
```
Comment (wanchaol) [inline test/benchmarks.py]: hmmm shouldn't this whole block be under the indent of the `with` block? I thought `prof` will only be visible inside the block

**Review by wanchaol — APPROVED** [review]

File: test/benchmarks.py (hunk 2 above)
Comment (bwasti) [reply test/benchmarks.py]: nope, with doesn't create a scope

**Pull request closed without merging**